tests/methods/test_maskers.py

```diff
@@ -10,37 +12,78 @@ def test_mask_has_correct_shape_univariate():
     input_data = _get_univariate_input_data()
     number_of_masks = 5
 
-    result = _call_masking_function(input_data, number_of_masks=number_of_masks)
+    result = _call_masking_function(input_data,
+                                    number_of_masks=number_of_masks)
```

Comment:
if the test only checks the shape, calling `generate_masks(input_data, number_of_masks=number_of_masks)` is enough instead of `_call_masking_function`